token-service/src/main/java/org/p2p/token/service/model/TokenServiceMetadata.kt

```diff
@@ -0,0 +1,9 @@
+package org.p2p.token.service.model
+
+data class TokenServiceMetadata(
+    val address: String,
+    val symbol: String,
+    val logoUrl: String,
+    val decimals: Int,
+    val price: TokenServicePrice
```

Comment:
not nullable? Is there any case that price cannot be loaded?

Reply:
For now yes, if I will find a case that price is null, I will change it